reflex/state.py

```diff
@@ -1192,7 +1204,8 @@ async def _process_event(
         # If an error occurs, throw a window alert.
         except Exception:
             error = traceback.format_exc()
-            print(error)
+            # print(error)
+            BaseState._handle_exception(error)
```

Comment:
You can try to use `prerequisites.get_app()` to get the reflex app instance and your exception handler here